runtime/runtime-params-estimator/emu-cost/ci.sh

```diff
@@ -14,7 +14,7 @@ docker run \
      /usr/bin/env bash -c "
 set -ex
 cd /host/${srcdir}/runtime/runtime-params-estimator
-$(cd test-contract && ./build.sh)
+: "'$(cd test-contract && ./build.sh)'"
```

Comment:
What is this magic and difference compare to `$()` :D I googled but this is hard to search

Reply:
@ailisp when you do `$(echo pwd)` you will get the `$()` resolved to `pwd`, and it will get executed:

![image](https://user-images.githubusercontent.com/304265/120365488-deba1080-c2e4-11eb-8a4d-b2f910684888.png)

So the current version of the script resolves `./build.sh` and then all the output is treated as a command that needs to be executed.

`:` (colon) is a built-in no-op command, so this trick basically will end up calling no-op with `pwd` argument, which is fine.

It is better to re-write this piece of code into something like:

```
pushed test-contract
./build.sh
popd
```

instead of crafting magic here :smile: